external/libwally-core/src/test/test_sign.py

```diff
@@ -135,7 +135,7 @@ def test_invalid_inputs(self):
                  (priv_key, len(priv_key),   out_buf, 10)]           # Wrong out len
 
         for pk, pk_len, o, o_len in cases:
-            ret = wally_ec_public_key_from_private_key(pk, pk_len, o, o_len);
+            ret = wally_ec_public_key_from_private_key(pk, pk_len, o, o_len)
```

Comment:
`external/libwally-core` is an external dependency and should not be edited here, but upstream.